app/models/grda_warehouse/hud/project.rb

```diff
@@ -601,21 +617,17 @@ def organization_name(user = nil)
       organization.OrganizationName
     end
 
-    def confidential_name?
-      confidential? || /healthcare/i.match(self.ProjectName).present?
-    end
-
-    def organization_and_name(include_confidential_names: false)
-      project_name = name(include_confidential_names: include_confidential_names, include_project_type: true)
-      return "#{organization&.OrganizationName} / #{project_name}" if include_confidential_names
+    def organization_and_name(user = nil, ignore_confidential_status: false)
+      project_name = name(user, include_project_type: true, ignore_confidential_status: ignore_confidential_status)
+      return "#{organization&.OrganizationName} / #{project_name}" if user.can_view_confidential_enrollment_details?
 
-      "#{organization&.OrganizationName} / #{project_name}" unless confidential_name?
+      "#{organization&.OrganizationName} / #{project_name}" unless confidential?
```

Comment:
I _think_ this needs a return
```suggestion
      return "#{organization&.OrganizationName} / #{project_name}" unless confidential?
```

Reply:
Yeah I suppose it does! I'll add. It looks like we don't use this in many place so the impact wont be huge. 
https://github.com/greenriver/hmis-warehouse/pull/2225/commits/95d6de73a1caf3f06516289017bea05df546f19b